client/app/app.scss

```diff
@@ -30,3 +30,177 @@ body,
 @import 'main/main.scss';
 @import 'modal/modal.scss';
 // endinjector
+
```

Comment:
Instead of putting this directly in app.scss, we should make a new scss file in /client/components/navbar. The grunt injector task should pick up the new file and add an import.